packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -425,6 +429,7 @@ class CheckboxListTile extends StatelessWidget {
     return MergeSemantics(
       child: ListTileTheme.merge(
         selectedColor: activeColor ?? Theme.of(context)!.accentColor,
+        tileColor: tileColor,
```

Comment:
Is there a reason in passing the tileColor into `ListTileTheme` here vs `ListTile` down below? If not, maybe we should just pass it directly into `ListTile` since that's where most of the parameters are defined. It seems like there might be a special reason that we have a `ListTileTheme` here for `selectedColor` and `accentColor`.

Reply:
> It seems like there might be a special reason that we have a `ListTileTheme` here for `selectedColor` and `accentColor`.

I thought the reason is to also change the colors of descedant ListTile widgets?! I'm not really sure. If this statement is not true then maybe I can just pass it directly into ListTile.